src/GitHub.Exports.Reactive/Services/IPullRequestService.cs

```diff
@@ -124,9 +124,10 @@ public interface IPullRequestService
         /// <summary>
         /// Gets the encoding for the specified file.
         /// </summary>
-        /// <param name="path">The path to the file.</param>
+        /// <param name="repository">The repository.</param>
+        /// <param name="relativePath">The relative path to the file in the repository.</param>
         /// <returns>The file's encoding</returns>
```

Comment:
Should mention behaviour if file doesn't exist (return `Encoding.Default` or throw?).